static/devtron.css

```diff
@@ -275,6 +275,7 @@ pre,
 code {
   color: inherit;
   background-color: inherit;
+  -webkit-user-select: text;
```

Comment:
Yup, this is great to have for code/pre blocks 👍  I think photon disables selection by default to be more native feeling.